kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/rocksdb/RocksDBStateStoreConfigSetter.java

```diff
@@ -1,64 +1,25 @@
 package org.hypertrace.core.kafkastreams.framework.rocksdb;
```

Comment:
instead of updating the existing RocksDBStateStoreConfigSetter, can we have a separate class using write buffer manager and add `deprecated` annotation to this class. this will allow us to migrate to new buffer manager in a controlled manner.

Reply:
Fixed. Now we have 2 config setter implementations and this can be overridden at config level.
We will be removing the old deprecated implementation in future versions